build-shaded-dtest-jar.sh

```diff
@@ -2,12 +2,17 @@
 
 set -xe
 
-# Cassandra version that is specified in build.xml
-CASSANDRA_VERSION=$1
-# In-JVM dtest version that is specified in relocate-dependencies.pom
-DTEST_VERSION=$2
+cp .//build/lib/jars/reflections-0.9.12.jar ./lib
+cp .//build/lib/jars/javassist-3.26.0-GA.jar ./lib
+
 ARTIFACT_NAME=cassandra-dtest
 REPO_DIR=~/.m2/repository
+CASSANDRA_VERSION=$(cat build.xml | grep 'property name="base.version"' | awk -F "\"" '{print $4}')
+DTEST_VERSION=$(cat relocate-dependencies.pom | grep "dtest-local.version>" | awk -F "\>|\<" '{print $3}')
```

Comment:
how are we expected to manage this version?  it seems decoupled from CASSANDRA-VERSION and doesn't actually show the dtest-api version; should we not just have this match the CASSANDRA_VERSIOn?

Reply:
true, this line doesn't get the DTEST_VERSION. As I was in a rush last night just to run it, I hardcoded the version on my own to make it run

Reply:
@dcapwell the point of this patch was specifically to avoid anyone to have to manage this version. I've changed it from `4.0.1-SNAPSHOT` to `4.0-beta4-SNAPSHOT`, since we'll need to publish it several times. Howver, if someone either forgets or has different values for the two by mistake, it'll still work.

Reply:
I won't block fixing this for this ticket, but would be good to have a single point of truth, as I feel that relocate-dependencies.pom will be neglected.